File: Arrays/Valid-Anagrams.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """

        if len(s) != len(t):
            return False

        arr = [0 for i in range(0,26)]
        index = 0
        while index < len(s):
            arr[ord(s[index])%26] += 1
            arr[ord(t[index])%26] -= 1
            index += 1
        
        for num in arr:
            if num > 0:
                return False

        return True

File: Arrays/Valid-Anagrams.java
import java.util.Arrays;

class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()){
            return false;
        }

        int[] arr = new int[26];

        int i = 0;
        while(i < s.length()){
            arr[(int)s.charAt(i)%26] += 1;
            arr[(int)t.charAt(i)%26] -= 1;  
            i++;         
        }
    
        if(Arrays.stream(arr).parallel().filter(a -> a!=0).findFirst().orElse(0) != 0){
            return false;
        }

        return true;
    }
}